simulation/simulation_workflow.ipynb

```diff
@@ -16,10 +16,32 @@
     "from PIL import Image\n",
     "from scipy import interpolate\n",
     "import matplotlib.pyplot as plt\n",
-    "\n",
```

Comment:
how about this one - is it changed?

Reply:
I did add a cell or two to run SLiM and `compute_stats.py` -- mostly in efforts to get my head around stuff. happy to revert.